web/modules/custom/dpl_base/dpl_base.info.yml

```diff
@@ -0,0 +1,5 @@
+name: "DPL Base"
+type: module
+description: "Generalized logic, such as enabling/disabling modules, migrating data or similar."
+package: DPL
```

Comment:
I second that. It is dangerous to name a module base or something to broad since it introduce that all kind of hacks or just features where people do not decide which module should be responsible for it.

Reply:
But I am not sure that I agree in using the profile for it.
Could it make sense to call the module dpl_webmaster_config (or just dpl_config) and add the config_ignore_auto as a dependency to that?